.appveyor.yml

```diff
@@ -1,31 +1,32 @@
 version: '{branch}.{build}'
 skip_tags: true
-image: Visual Studio 2017
+image: Visual Studio 2019
 configuration: Release
 platform: x64
 clone_depth: 5
 environment:
   APPVEYOR_SAVE_CACHE_ON_ERROR: true
-  CLCACHE_SERVER: 1
-  PACKAGES: berkeleydb boost-filesystem boost-signals2 boost-test libevent openssl rapidcheck zeromq double-conversion
+  PACKAGES: berkeleydb boost-filesystem boost-multi-index boost-signals2 boost-test boost-thread libevent openssl rapidcheck zeromq double-conversion
   PATH: 'C:\Python37-x64;C:\Python37-x64\Scripts;%PATH%'
   PYTHONUTF8: 1
-  QT_DOWNLOAD_URL: 'https://github.com/sipsorcery/qt_win_binary/releases/download/v1.0/Qt5.9.7_ssl_x64_static_vs2017.zip'
-  QT_DOWNLOAD_HASH: 'D4D35B8112302B67E5610A03421BB3E43FE13F14D9A5F637C22AE60DCEC0E0F5'
-  QT_LOCAL_PATH: 'C:\Qt5.9.7_ssl_x64_static_vs2017'
+  QT_DOWNLOAD_URL: 'https://github.com/sipsorcery/qt_win_binary/releases/download/v1.4/Qt5.9.8_x64_static_vs2019.zip'
+  QT_DOWNLOAD_HASH: 'f285cbb02bec3b3f3cc2621e3fa7d5edf0d6a66fa30c57859e583acda954ea80'
+  QT_LOCAL_PATH: 'C:\Qt5.9.8_x64_static_vs2019'
 cache:
 - C:\tools\vcpkg\installed
-- C:\Users\appveyor\clcache -> .appveyor.yml, build_msvc\**, **\Makefile.am, **\*.vcxproj.in
-- C:\Qt5.9.7_ssl_x64_static_vs2017
+- C:\Qt5.9.8_x64_static_vs2019
 install:
-- cmd: pip install --quiet git+https://github.com/frerich/clcache.git@v4.2.0
 # Disable zmq test for now since python zmq library on Windows would cause Access violation sometimes.
 # - cmd: pip install zmq
 - cmd: echo set(VCPKG_BUILD_TYPE release) >> C:\tools\vcpkg\triplets\%PLATFORM%-windows-static.cmake
+- cmd: cd c:\tools\vcpkg
```

Comment:
You mentioned [here](https://github.com/bitcoin/bitcoin/pull/17309#issuecomment-548083497) that updating `vcpkg` for each build would be infeasible. Is that no-longer the case, or am I mis-understanding what this does?

Reply:
I checked my statement by running the appveyor job with a pull of the vcpkg repo as mentioned in the articles you [referenced](https://www.appveyor.com/docs/lang/cpp/#vc-packaging-tool) and I was incorrect about the time it would take. 

The reason being that updating and rebuilding the `vcpkg` program does not affect already installed packages. In my testing it only added about 2 minutes onto the build time. I wrongly assumed that updating vcpkg would invalidate the existing packages and force them to be re-installed.

For this PR the `vcpkg` source does have to be updated to get the latest port files otherwise the build fails (didn't track down why). After the first successful build the`vcpkg` update steps can be disabled to save the 2 minutes. It would need to be some kind of conditional disable so that anyone forking the repo still gets a working appveyor config. I can look into it if consensus is the small additional complexity justifies saving 2 minutes.